gustaf/create/faces.py

```diff
@@ -0,0 +1,99 @@
+"""gustaf/create/faces.py
+
+Routines to create faces.
+"""
+
+import numpy as np
+
+from gustaf.faces import Faces
+from gustaf.utils import log
```

Comment:
```suggestion
from gustaf import utils
```
With exception of single function import, I think I've been keeping full namespace. So id gently ask here too :)